src/main/UI/TextField.java

```diff
@@ -0,0 +1,20 @@
+package UI;
+
+import javax.swing.*;
+
+public class TextField {
+    JTextField textField;
+
+    public TextField() {
+    }
+
```

Comment:
I think this can be package-private rather than public?

Reply:
Yes, I will get rid of constructors that are unused.